composeApp/src/commonMain/kotlin/com/revenuecat/purchases/kmp/sample/components/PackageRow.kt

```diff
@@ -0,0 +1,45 @@
+package com.revenuecat.purchases.kmp.sample.components
+
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.padding
+import androidx.compose.material.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import com.revenuecat.purchases.kmp.Package
+import com.revenuecat.purchases.kmp.sample.DefaultPaddingHorizontal
+
+@Composable
+internal fun PackagesRow(
+    pkgs: List<Package>,
+    label: String,
+    modifier: Modifier = Modifier,
+) {
+    CollapsibleRow(
+        modifier = modifier,
+        collapsedContent = { Text(text = "$label: ${pkgs.size}") },
+        expandedContent = {
+            Column(modifier = Modifier.padding(start = DefaultPaddingHorizontal)) {
+                pkgs.forEach { pkg -> PackageRow(pkg = pkg) }
+            }
+        },
+    )
+}
+
+@Composable
+internal fun PackageRow(
+    pkg: Package,
+    modifier: Modifier = Modifier,
+    label: String = "package"
+) {
+    CollapsibleRow(
+        collapsedContent = { Text(text = "$label: ${pkg.identifier}") },
+        expandedContent = {
+            Column(modifier = Modifier.padding(start = DefaultPaddingHorizontal)) {
```

Comment:
Could be nice to add buttons for purchasing the objects that can be purchased (package/subscriptionOption/storeProduct) with the appropriate purchase method. And other utility buttons to test other functions as well.... But I think it can happen in future PRs too!

Reply:
Yea I think that's useful, but indeed probably better in a separate PR.